specs/eventing/data-plane.md

```diff
@@ -1,125 +1,183 @@
+reliable event delivery and event-driven applications can be built.
 
-### Content Modes Supported
+## Background
 
-A Sink MUST support `Binary Content Mode` and `Structured Content Mode` as
-described in
-[HTTP Message Mapping section of HTTP Protocol Binding for CloudEvents](https://github.com/cloudevents/spec/blob/master/http-protocol-binding.md#3-http-message-mapping)
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in RFC2119.
 
-A Sink MAY support `Batched Content Mode` but that mode is not used in Knative
-Eventing currently (that may change in future).
+When not specified in this document, the
+[CloudEvents HTTP bindings, version 1.0](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md)
+and [HTTP 1.1 protocol](https://tools.ietf.org/html/rfc7230) standards should be
+followed (with the CloudEvents bindings taking precedence in the case of
+conflict).
 
-### Retries
+The current version of this document does not describe protocol negotiation or
+any delivery mechanism other than HTTP 1.1. Future versions might define
+protocol negotiation to optimize delivery; compliant implementations SHOULD aim
+to interoperate by ignoring unrecognized negotiation options (such as
+[HTTP `Upgrade` headers](https://datatracker.ietf.org/doc/html/rfc7230#section-6.7)).
 
-Sinks should expect that retries and accept possibility that duplicate events
-may be delivered.
+## Event Delivery
 
-### Error handling
+### Minimum supported protocol
 
-If Sink is not returning HTTP success header (200 or 202) then the event may be
-sent again. If the event can not be delivered then some sources of events (such
-as Knative sources, brokers or channels) MAY support
-[dead letter sink or channel](https://github.com/knative/eventing/blob/main/docs/delivery/README.md) for events that can not be
-delivered.
+All senders and recipients MUST support the CloudEvents 1.0 protocol and the
+[binary](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#31-binary-content-mode)
+and
+[structured](https://github.com/cloudevents/spec/blob/v1.0.1/http-protocol-binding.md#32-structured-content-mode)
+content modes of the CloudEvents HTTP binding. Senders which do not advertise
+the ability to accept [reply events](#derived-reply-events) MAY implement only
+one content mode, as the recipient is not allowed to negotiate the content mode.
+Senders MUST support both cleartext (`http`) and TLS (`https`) URLs as event
+delivery destinations.
+
+### HTTP Verbs
+
+In the absence of specific delivery preferences, the sender MUST initiate
+delivery of the event to the recipient using the HTTP POST verb, using either
+the structured or binary encoding of the event (sender's choice). This delivery
+SHOULD be performed using the CloudEvents HTTP Binding, version 1.0.
```

Comment:
any reason this isn't a MUST? Or is it a SHOULD to go along with the "delivery preference" stuff above?  If it's because of the SHOULD then I think it should be a MUST with the proper wiggle wording like the first sentence has.

Reply:
I'd prefer a MUST, but we could just say "v1.x" of the CE specs, since a v2.0 would probably require a major change for us anyway.